### PR TITLE
reformated dates from "1994-06-30T00:00:00 to Thursday, June 30, 1994"

### PR DESCRIPTION
**Pull request by 308butterfly**

### CODE REVIEW

**Review by escalonn — CHANGES_REQUESTED** [review]
like i've already said, you have to look at the 'files changed' tab (the diff) yourself before requesting review.

**Comment by 308butterfly** [comment]
Changed some settings in Prettier in Visual Studio Code.  Will have to be extremely vigilant about this.  Should be okay now

File: fakebook/src/app/component/newsfeed/newsfeed.component.html [hunk 1]
```diff
@@ -25,15 +25,12 @@ <h3>{{ user?.firstName }} {{ user?.lastName }}</h3>
...
+        <app-post-view [post]="post" [userEmail]="user.email" (notifyComment)="onNotifyComment($event)">
+        </app-post-view>
       </div>
     </div>
     <div class="col-md-6">
       <!--Insert followers component here-->
     </div>
   </div>
-</div>
+</div>
```
Comment (escalonn) [inline fakebook/src/app/component/newsfeed/newsfeed.component.html:36]: these little changes like removing the newline at the end of files do not belong.

**Review by escalonn — APPROVED** [review]
it can be difficult to take full control of these editor settings but it is necessary. you're responsible for justifying every single line diff that appears in this "files changed" tab, whether you did it or a tool you were using did it, every time you make a review request. there's a lot of unrelated changes here - this PR _should_ be a change to exactly 2 lines and no more. this might seem overly particular but it really is how a lot of teams work.

the remaining formatting changes aren't such a big deal, so i'll approve this, and let you or your team lead decide whether to continue the effort or just merge it.

**Pull request closed without merging**